CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v4.4.1 - 2023-10-17
```

Comment:
you might wanna mention https://github.com/octoenergy/xocto/commit/98555998e401100cd56d1773900afa892b78eb1e as well?

Reply:
👍  Thanks for flagging, I had a quick look for other changes but I didn't see a merge commit for that PR so overlooked the change.

As a result a I have changed the version bump to `v4.5.0` instead as this `strftime` is actual new functionality.